src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/MS/Internal/MarkupCompiler/NETFrameworkGetRelativePath.cs

```diff
@@ -0,0 +1,380 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//---------------------------------------------------------------------------
+//
+// Description:
+//   Returns a relative path from one path to another.  
+//
+//   Paths are resolved by calling the GetFullPath method before calculating 
+//   the difference. The method uses the default file path comparison for the 
+//   current platform (StringComparison.OrdinalIgnoreCase for Windows.)
+//
+//---------------------------------------------------------------------------
+
+#pragma warning disable 1634, 1691
+
+using System;
+using System.Xml;
+using System.IO;
+using System.Text;
+using System.Reflection;
+using System.Globalization;
+using System.ComponentModel;
+using System.Security.Cryptography;
+
+using System.CodeDom;
+using System.CodeDom.Compiler;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel.Design.Serialization;
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+
+using System.Threading;
+using MS.Internal.Markup;
+using MS.Internal.Tasks;
+using MS.Utility;   // for SR
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Tasks.Windows;
+using System.Runtime.CompilerServices;
+
+namespace MS.Internal
+{
+    internal sealed class PathInternal 
+    {
+        internal static string GetRelativePath(string relativeTo, string path, StringComparison comparisonType)
+        {
+            if (relativeTo == null)
+                throw new ArgumentNullException(nameof(relativeTo));
+
+            if (PathInternal.IsEffectivelyEmpty(relativeTo.AsSpan()))
+                throw new ArgumentException(nameof(relativeTo));
```

Comment:
The constructor of ArgumentException with one argument is a message not a parameter name

```
public ArgumentException (string message)
```

https://docs.microsoft.com/zh-cn/dotnet/api/system.argumentexception.-ctor?view=netcore-3.1#System_ArgumentException__ctor_System_String_

Reply:
tip: If you have more than one review comment to offer, start a review instead of adding a comment. That way it doesn't pollute the PR with unnecessary breaks.

Reply:
Thanks, @lindexi.  This is standard convention for the .NET Core runtime.  Let me know if you disagree and I'll reactive.

Reply:
@ryalanms Sorry, I do not know it is standard convention for the .NET Core runtime. But I think if the parameter name is used as the message, then it may be difficult for us to get enough information from the message. We can only learn which parameter is wrong from the message, but we don’t know the reason for the error.